src/universal/redux/segmentActions.js

```diff
@@ -1,22 +1,75 @@
 import {EventTypes} from 'redux-segment';
+import {cashay} from 'cashay';
+import {getAuthQueryString, getAuthedOptions} from './getAuthedUser';
 
 /**
  * Sometimes, we just want to track events in segment.
  */
 
 const SEGMENT_EVENT = '@@segment/EVENT';
 
-export function segmentEvent(event, properties) {
-  return {
-    type: SEGMENT_EVENT,
-    meta: {
-      analytics: {
-        eventType: EventTypes.track,
-        eventPayload: {
-          event,
-          properties
+const defaultProfile = {
+  avatar: null,
+  email: null,
+  id: null,
+  name: null
+};
+
+export function selectSegmentProfile(state) {
+  if (!state.auth || !cashay.store) { return defaultProfile; }
```

Comment:
I think this is safe to remove. If we don't have both of these, we have a lot bigger problems!